check_emoji_sequences.py

```diff
@@ -389,25 +395,36 @@ def collect_name_to_dirpath_with_override(dirs, prefix, suffix, exclude=None):
   return result
 
 
-def run_check(dirs, prefix, suffix, exclude, unicode_version, coverage):
+def run_check(dirs, names, prefix, suffix, exclude, unicode_version, coverage):
   msg = ''
   if unicode_version:
     msg = ' (%3.1f)' % unicode_version
-  print(f'Checking files with prefix "{prefix}" and suffix "{suffix}"{msg} in: {dirs}')
-  name_to_dirpath = collect_name_to_dirpath_with_override(
-      dirs, prefix=prefix, suffix=suffix, exclude=exclude)
+  
+  if (names and dirs):
+    sys.exit("Please only provide a directory or a list of names")
+  elif names:
+    name_to_dirpath = {}
+    for name in names:
+      name_to_dirpath[name] = ""
+  elif dirs:
+      print(f'Checking files with prefix "{prefix}" and suffix "{suffix}"{msg} in: {dirs}')
```

Comment:
indentation in the rest of the module is 2 spaces, here you used 4 (I don't like that either, but that's how it is right now)

Reply:
Ahh good catch, will fix it!